html/semantics/forms/form-submission-0/resources/form-submission.py

```diff
@@ -1,8 +1,9 @@
+from six import ensure_str
 def main(request, response):
-    if request.headers.get('Content-Type') == 'application/x-www-form-urlencoded':
-        result = request.body == 'foo=bara'
-    elif request.headers.get('Content-Type') == 'text/plain':
-        result = request.body == 'qux=baz\r\n'
+    if ensure_str(request.headers.get('Content-Type')) == 'application/x-www-form-urlencoded':
```

Comment:
Wouldn't it be easier to use b"" here (and below) on the other side?

Reply:
Shouldn't this kind information be string?

Reply:
Why?